experimental/traffic-portal/src/app/core/topologies/topology-details/topology-details.component.html

```diff
@@ -0,0 +1,44 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<mat-card appearance="outlined" class="page-content single-column">
+	<tp-loading *ngIf="loading"></tp-loading>
+	<form ngNativeValidate (ngSubmit)="submit($event)">
+		<mat-card-content class="container">
+			<mat-form-field>
+				<mat-label>Name</mat-label>
+				<input matInput type="text" name="name" required [value]="topology.name">
+			</mat-form-field>
+			<mat-form-field>
+				<mat-label>Description</mat-label>
+				<textarea matInput name="description" [value]="topology.description"></textarea>
```

Comment:
`[value]=...` is a one-way data binding. The user can type in these fields, but what is submitted in the subsequent `PUT` or `POST` request is whatever was set up behind the scenes at pageload, because these controls cannot be used to modify that data. Use `[(ngModel)]=...` instead

Reply:
Note that this means that even if there were controls for manipulating Topology nodes, a Topology cannot be created because it would always have a blank name, which isn't allowed.

Reply:
Using `[(ngModel)]` in 4fbced122d